test/PHPUnit/Controller/AbstractConsoleControllerTestCaseTest.php

```diff
@@ -39,10 +39,8 @@ public function testAssertResponseStatusCode()
         $this->dispatch('--console');
         $this->assertResponseStatusCode(0);
 
-        $this->setExpectedException(
-            ExpectationFailedException::class,
-            'actual status code is "0"' // check actual status code is display
-        );
+        $this->expectException(ExpectationFailedException::class);
+        $this->expectExceptionMessage('actual status code is "0"' /* check actual status code is display */ );
```

Comment:
Can we have these comments after the statement, like that:

```php
$this->expectExceptionMessage('actual status code is "0"'); // check actual status code is display
```

... if we need them at all.